chart/templates/deployment.yaml

```diff
@@ -10,6 +10,12 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
   {{- include "chart.labels" . | nindent 4 }}
+{{- with .Values.controller.imagePullSecrets }}
+imagePullSecrets:
+{{- range . }}
+  - name: {{ .name }}
+{{- end }}
+{{- end }}
```

Comment:
We may want to consider also supporting just a list of just the secret names as well, similar to https://github.com/hashicorp/vault-helm/pull/576/files

i.e.
```yaml
controller:
  imagePullSecrets:
    - pull-secret-name
    - pull-secret-name-2
```

Reply:
Good call, I did see that is how Vault-helm did things, but was on the fence about it, definitely can standardize on it!

Reply:
Fixed in 59822c9b352bacd25200fcbffc3f3620d462a3b8